pkg/reconciler/eventtransform/resources_jsonata.go

```diff
@@ -0,0 +1,176 @@
+package eventtransform
```

Comment:
nit: we could license header? and perhaps stash that file in a separate `resources` folder? Like others do?

Reply:
Added